_data/simple-icons.json

```diff
@@ -1730,6 +1730,11 @@
             "hex": "1428A0",
             "source": "https://www.samsung.com/us"
         },
+        {
+            "title": "Samsung Pay",
+            "hex": "3356D0",
```

Comment:
Where did you get this hex value from? The [Full Guide](https://d3sfvyfh4b9elq.cloudfront.net/ptr/document_guide/75800910295990609/SamsungPay_Guidelines_FINAL_2018.pdf) doesn't mention this color at all 🤔

Reply:
I download the Android app apk, extract the icon, and use an eyedropper on figma. Now that i think of it, I might even be able to find the actual color code form the resource in the apk itself. 1 sec......

Reply:
Can't figure out how to get the color code back out, but I did find [this section of their website](https://www.samsung.com/hk_en/samsungpay/#simple-to-use) which has `linear-gradient(135deg, #0140b4 53%,#0263da 100%)` as the background color. Are those gonna be more appropriate?

Reply:
From the guidelines I linked above:

> Samsung Pay uses a streamlined version of the masterbrand color palette, divided into two color groups: primary colors and secondary colors. 

Hence I think we should use any of the colors specified there, probably `#1428A0`